src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TovStar.hpp

```diff
@@ -0,0 +1,317 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/PolytropicFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;  // IWYU pragma: keep
+}  // namespace PUP
+/// \endcond
+
+// IWYU pragma: no_include <pup.h>
+
+namespace RelativisticEuler {
+namespace Solutions {
+
+/*!
+ * \brief A static spherically symmetric star
+ *
+ * An analytic solution for a static spherically symmetric star found by
+ * solving the Tolman-Oppenheimer-Volkoff (TOV) equations.  The equation of
+ * state is assumed to be that of a polytropic fluid.
+ *
+ * \tparam RadialSolution selects not only how the TOV equations are solved, but
+ * also the radial coordinate that the solution is given in (e.g. areal or
+ * isotropic).  See the documentation of the specific `RadialSolution` for
```

Comment:
`RadialSolution` -> `RadialVariables`

Reply:
no, I mean `RadialSolution`; I'll try to clarify